### PR TITLE
Taiga

### PR DESCRIPTION
**Pull request by wuxmedia**
taiga and tutanota added, possibly

### CODE REVIEW

**Comment by wuxmedia** [comment]
OMG no conflicts.

**Comment by alexanderadam** [comment]
If I may say that: you did some great contributions! :+1:

**Comment by wuxmedia** [comment]
could this be accepted? I've caught a bit of a minimal SVG bug.

**Comment by edent** [comment]
I'll take a look this weekend. Thank you for all your contributions 😁

**Comment by wuxmedia** [comment]
sure - sorry I sounded a bit pushy! I was just making sure I hadn't made some sort of mistake :D

File: images/svg/tutanota.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg
+xmlns="http://www.w3.org/2000/svg"
+aria-label="..." role="img"
```
Comment (edent) [inline images/svg/tutanota.svg]: This ARIA label needs to be completed.

File: images/svg/tutanota.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+viewBox="0 0 512 512"><rect
+width="512" height="512"
+rx="15%"
+fill="#fff"/>
+<path d="M113 97c-17 0-31 14.09-31.455 31.451v290.461c0 1.102.063 2.212.17 3.284a1700 1700 0 0037-14c91-36 165-67 165-102 0-1.12-.077-2.25-.236-3.38-4.717-34.512-87.191-45-87-61.028.01-.84.248-1.719.754-2 9-17 49-16 63-17 14-1 48-.996 50-11.336.048-.32.078-.637.078-.956.039-9-23-13-23-13s28 4 28 15c0 .54-.066 1.1-.217 1.67-3.055 11-28 14-44 14-15.615.782-39 2-39 10.186-.01.445.067.908.226 1 3 11 90 16 146 45 32 16 48 44 55 73V128c0-17-14-31-31-31H113z" fill="#a01e20"/></svg>
```
Comment (edent) [inline images/svg/tutanota.svg]: Remove this newline.

File: images/svg/taiga.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg
+xmlns="http://www.w3.org/2000/svg"
+aria-label="taiga.io" role="img" 
+viewBox="0 0 512 512"><rect
+width="512" height="512"
+rx="15%"
+fill="#fff"/>
+<path d="M393 119l-17 119L256 256l17-120z" opacity=".8" fill="#a295ae"/>
```
Comment (edent) [inline images/svg/taiga.svg]: Remove all these newlines to shrink the file.

File: images/reference/taiga.svg [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (edent) [inline images/reference/taiga.svg]: GitHub gives an error `Sorry, this file is invalid so it cannot be displayed.`

File: images/reference/tutanota.svg [hunk 5]
```diff
@@ -0,0 +1,85 @@
+￼<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (edent) [inline images/reference/tutanota.svg]: GitHub gives an error `Sorry, this file is invalid so it cannot be displayed.`

File: images/svg/tutanota.svg (hunk 2 above)
Comment (wuxmedia) [reply images/svg/tutanota.svg]: which line? (number)

File: images/svg/taiga.svg (hunk 3 above)
Comment (wuxmedia) [reply images/svg/taiga.svg]: again, which lines? I thought that was supposed to be the head.

Comment (wuxmedia) [reply images/svg/taiga.svg]: ok I got rid of the newlines. just made it possible to edit without losing my mind :D

File: images/svg/tutanota.svg (hunk 2 above)
Comment (edent) [reply images/svg/tutanota.svg]: At the end of line 7. `fill="#fff"/>\n<path` can become `fill="#fff"/><path`

**Comment by wuxmedia** [comment]
do I make the readme up to the current then?

File: images/svg/tutanota.svg (hunk 2 above)
Comment (wuxmedia) [reply images/svg/tutanota.svg]: ok done.

**Comment by wuxmedia** [comment]
i don't know how to resolve this conflict.

**Comment by jmb** [comment]
It should show you the conflicting lines when you click into the conflict. It's probably just the fact that another icon has been added to the README in the same place as yours - you can probably accept both changes and put your icon on the line below. This might help explain: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github

**Comment by wuxmedia** [comment]
I understand now. didn't know one could edit that file. I think I diverged from the master file though.
oh well

File: README.md [hunk 6]
```diff
@@ -177,6 +178,9 @@ Say thanks!
 <td>Sentry<br><img src="https://edent.github.io/SuperTinyIcons/images/svg/sentry.svg" width="125" title="Sentry" /><br>507 Bytes</td>
 <td>Behance<br><img src="images/svg/behance.svg" width="125" title="Behance" /><br>916 Bytes</td>
 </tr>
+<tr>
```
Comment (jmb) [inline README.md]: You probably want to remove the `</tr>` and `<tr>` here as there is still space in the row above.

**Comment by jmb** [comment]
I think this looks ok (in terms of resolving the conflict). You might want to put the Taiga icon in the readme in the previous row rather than starting a new one though as there is space - see the comment I've put in the change!

File: README.md (hunk 6 above)
Comment (wuxmedia) [reply README.md]: ok thanks! didn't see that :D

**Comment by wuxmedia** [comment]
so how would I avoid having conflicts due updates made to readme.md (other commits coming it)? perhaps update my local repo to current ? Or just hack the file and merge?

**Comment by jmb** [comment]
Yes, you either have to update your repo from the master, or just use the merge tools. I don't know "the best" way - I'm just a hobbyist and tend to fumble my way around git!

**Comment by edent** [comment]
@wuxmedia You currently have `tutanota.svg` saying `aria-label="..."`

You need to change that to `aria-label="Tutanota"` and then update the filesize in the README.

See https://raw.githubusercontent.com/edent/SuperTinyIcons/47173d1a4b8260cc07184bd4577bacae86f328b5/images/svg/tutanota.svg

I'll merge once that is done. Cheers :-)

**Comment by wuxmedia** [comment]
ok Done!

**Comment by edent** [comment]
All done. In future, probably best to add only one icon per PR.

**Pull request merged**